ffi/src/schema.rs

```diff
@@ -63,6 +65,7 @@ pub struct EngineSchemaVisitor {
         data: *mut c_void,
         sibling_list_id: usize,
         name: KernelStringSlice,
+        is_nullable: bool,
         value_contains_null: bool, // if this map can contain null values
```

Comment:
```suggestion
        is_nullable: bool,
```
(see commentary about the TODO flagged below)